### DISCUSSION TITLE
Is the optimize.print_gain option applied to the pngquant utility?

### DISCUSSION THREAD

**Discussion by AnastasiaTWW** [squidfunk/mkdocs-material#6454] [Q&A] [answered]
Is this option https://squidfunk.github.io/mkdocs-material/plugins/optimize/#config.print_gain expected to be applied to the pngquant output? I've set it to false but the number of bytes gained after optimizing each file is printed. If not, probably it should be mentioned in the documentation.

**Comment by squidfunk** [comment]
Could you please attach a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/)?

**Reply by AnastasiaTWW** [reply]
[9.5.0+insiders.4.46.0-optimize_print_results.zip](https://github.com/squidfunk/mkdocs-material/files/13602251/9.5.0%2Binsiders.4.46.0-optimize_print_results.zip)

If you run this reproduction for the first time (either mkdocs serve or mkdocs build), you get `INFO    -  Optimized media file: assets/images/favicon.png 1.9 kB` in the output. It even does not depend on the pngquant existance. I would expect not to get these messages with `print_gain: false` (especially when there are many images like 100-200).

**Reply by squidfunk** [reply, marked as answer]
Ah I see. No, so [`print_gain`](https://squidfunk.github.io/mkdocs-material/plugins/optimize/#config.print_gain) doesn't disable the log message, but it only disables that computation of the gain, i.e., `↓ 5%`. I believe the log messages are useful to show what the plugin does, i.e., that it actually compressed the images. We could in theory make logging configurable, but I'm not sure why? From our docs:

> Use this setting to control whether the plugin should print the number of bytes gained after optimizing each file. If you want to disable this behavior, use:

**Reply by AnastasiaTWW** [reply]
I see now. In our case the problem is that we have many images in our project and they all are listed in the build output. This is mostly the limitation of netlify as they do not allow to reuse the content (the .cache directory in this case) from the previous builds out of the box. That is why each new build of the documentation is accompanied by the long-long list of images that have been compressed during the build.

**Reply by squidfunk** [reply]
Well, we can allow to disable logging in the future, but it is not a priority. We have other things on our plate right now, like the new search, any bugs that pop up after releasing the latest features from the funding goal, the projects plugin, and helping users like you ☺️ We hope to be able to add more people to our team with increased funding.

**Reply by AnastasiaTWW** [reply]
Yes, of course I understand. Still huge thanks for your help! Should we close this discussion or not? Probably, others can have the same conditions and vote for that
